controllers/nodemaintenance_controller.go

```diff
@@ -208,9 +227,13 @@ func (r *NodeMaintenanceReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 
 	if err = drain.RunNodeDrain(drainer, nodeName); err != nil {
 		r.logger.Info("Not all pods evicted", "nodeName", nodeName, "error", err)
+		// maintenance in progress - some pods haven't been evicted
+		utils.NormalEvent(r.Recorder, nm, utils.EventReasonEvictingPods, utils.EventMessageEvictingPods)
```

Comment:
Nit: I'm not sure it's a good idea to omit this event.
IIUC this sort of event might omit several times and "pollute" the event stack, a log message might be enough here.

Reply:
It could "pollute" the event stack, even though I don't expect it to happen too many times per CR.
Originially I though it could signal the maintenance is still in progrees, but a user could view the CR status (e.g. `DrainProgress`) for a better indication.